lib/src/flagged/flagged_bloc.dart

```diff
@@ -110,6 +106,7 @@ class FlaggedBloc extends Bloc<FlaggedEvent, FlaggedState> {
     final List<int> dateMakerIds = List();
     final Context context = Context();
     final List<int> messageIds = List.from(await context.getChatMessages(Chat.typeStarred, Context.chatListAddDayMarker));
+    _messageListRepository.putIfAbsent(ids: messageIds.where((id) => id != ChatMsg.idDayMarker).toList());
```

Comment:
This is important, as all later performed requests depend on the existence of the messages within the repository. If one id is added all consumers can later load the whole message or just parts of it (dynamically via async calls from the core), but it's necessary to add the ids once, so the repository know which ids are actually valid. Without this call the "all flagged messages list" was empty for me.